### ISSUE TITLE
[website] Broken links to some download packages

### ISSUE THREAD

**Issue by Jennifer88huang** [apache/pulsar#5493]
**Describe the bug**
When I check the Pulsar website with link check tool, I've found some broken links to download packages. I've fixed the ones i could. Could someone give some help on the ones with red boxes? The following is the report.
![image](https://user-images.githubusercontent.com/47805623/67756694-6be81400-fa75-11e9-9e3d-f66fefb33ad0.png)

**Comment by Jennifer88huang** [comment]
@tuteng could you give some help? Thank you.

**Comment by tuteng** [comment]
I think this css file https://github.com/cckellogg/incubator-pulsar/blob/26d7dfbc651353f91eea4ecf0400685c331eaf5c/site2/website/static/css/code-blocks-buttons.css  can be added, but some download links I tested are normal.
https://archive.apache.org/dist/incubator/pulsar/pulsar-1.20.0-incubating/apache-pulsar-1.20.0-incubating-src.tar.gz
https://archive.apache.org/dist/incubator/pulsar/pulsar-1.19.0-incubating/apache-pulsar-1.19.0-incubating-src.tar.gz

But some file names are wrong, I think you can try to fix it.

![image](https://user-images.githubusercontent.com/1907867/67838808-26871d80-fb2d-11e9-84d5-42b859a56bfd.png)

**Comment by Jennifer88huang** [comment]
Yes, when you click https://archive.apache.org/dist/incubator/pulsar/pulsar-1.19.0-incubating/apache-pulsar-1.19.0-incubating-src.tar.gz, it works; however, when you click `sha512` under that package, it does not work. The `asc` links work well.
